pkg/scheduler/objects/application.go

```diff
@@ -1451,7 +1451,7 @@ func (sa *Application) notifyRMAllocationAskReleased(rmID string, released []*Al
 
 func (sa *Application) IsAllocationAssignedToApp(alloc *Allocation) bool {
 	sa.RLock()
-	defer sa.RLock()
+	defer sa.RUnlock()
```

Comment:
This change is already in master. I wonder why it still shows up as a diff?

Reply:
I think maybe because the PR branch didn't have master merged back in.